Project.toml

```
@@ -16,8 +16,8 @@ t8code_jll = "4ee9bed8-4011-53f7-90c2-22363c2f500d"
 [compat]
 CEnum = "0.4, 0.5"
 Libdl = "1"
-MPI = "0.20"
-MPIPreferences = "0.1.3"
+MPI = "0.20.6"
+MPIPreferences = "0.1.4"
```

```suggestion
MPIPreferences = "0.1.6"
```
cf. https://github.com/JuliaParallel/MPI.jl/blob/v0.20.6/Project.toml#L21